ref/general/health-check-microservices.adoc

```diff
@@ -25,15 +25,15 @@ A self-check can assess anything that the service needs:
 
 Services report as either `UP` (available) or `DOWN` (unavailable) by implementing the API provided by MicroProfile Health. A service orchestrator can then use these status reports to decide how to manage and scale the services within an application. Health checks can also interact directly with link:https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/[Kubernetes liveness and readiness probes].
 
-For example, in a microservices-based banking application, health checks might be implemented on the login service, the balance transfer service, and the bill pay service. If a health check on the balance transfer service detects a bug or deadlock, it reports as DOWN. In this case, if the `/health/live` endpoint is configured in a Kubernetes liveness probe, the probe restarts the service automatically, saving the user from encountering downtime or an error and preserving the functions of the other services in the application.
+For example, in a microservices-based banking application, you might implement health checks on the login service, the balance transfer service, and the bill pay service. If a health check on the balance transfer service detects a bug or deadlock, it reports as DOWN. In this case, if the `/health/live` endpoint is configured in a Kubernetes liveness probe, the probe restarts the service automatically. This feature saves the user from encountering downtime or an error and preserves the functions of the other services in the application.
 
 == MicroProfile Health provided endpoints and annotations
 
-MicroProfile Health provides three endpoints:
+MicroProfile Health provides the following three endpoints:
```

Comment:
I think this makes the sentence unnecessarily wordy. Having a colon at the end signals the start of a list of the three endpoints so I don't think this adds anything even with a screenreader.